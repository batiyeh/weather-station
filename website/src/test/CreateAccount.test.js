import React from 'react'
import CreateUserForm from '../components/registration/createUserForm'
import renderer from 'react-test-renderer';

it('renders correctly', () => {
    const tree = renderer
    .create(<CreateUserForm/>).toJSON();
    expect(tree).toMatchSnapshot();
})