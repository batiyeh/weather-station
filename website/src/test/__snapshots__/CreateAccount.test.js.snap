// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="register-container"
>
  <h2
    className="login-title"
  >
    Create Account
  </h2>
  <form
    action="/api/user/create/"
    id="createForm"
    method="post"
  >
    <div
      className="login-info mb-3"
    >
      <div
        className="form-group"
      >
        <input
          className="form-control"
          id="username"
          name="username"
          placeholder="Username"
          type="text"
        />
      </div>
      <div
        className="form-group"
      >
        <input
          aria-label="Email"
          className="form-control"
          id="email"
          name="email"
          placeholder="Email"
          type="email"
        />
      </div>
      <div
        className="form-group"
      >
        <input
          aria-label="Password"
          className="form-control"
          id="password"
          name="password"
          placeholder="Password"
          type="password"
        />
      </div>
    </div>
    <div
      className="row"
    >
      <div
        className="col-6 no-padding-left"
      >
        <a
          className="return-link"
          href="/user/login"
        >
          <button
            className="btn btn-default btn-block return-btn"
            type="button"
          >
            Return
          </button>
        </a>
      </div>
      <div
        className="col-6 no-padding-right"
      >
        <button
          className="btn btn-default btn-block"
          onClick="created()"
          type="submit"
        >
          Submit
        </button>
      </div>
    </div>
  </form>
</div>
`;
